src/phase0/presets/mainnet.rs

```diff
@@ -99,7 +99,7 @@ pub const PRESET: Preset = Preset {
 
 pub type IndexedAttestation = operations::IndexedAttestation<MAX_VALIDATORS_PER_COMMITTEE>;
 pub type PendingAttestation = operations::PendingAttestation<MAX_VALIDATORS_PER_COMMITTEE>;
-pub type HistoricalBatch = beacon_state::HistoricalBatch<SLOTS_PER_HISTORICAL_ROOT>;
+pub type HistoricalBatchAccumulator = beacon_state::HistoricalBatchAccumulator;
```

Comment:
we don't need this type defn anymore!

the idea w/ the types in the various `presets` is just to "specialize" them so users of this library don't see the generic parameters.

if we move ahead w/ this new accumulator type then we can just delete this line!